client/src/AppBundle/Resources/views/Report/Decision/start.html.twig

```diff
@@ -19,21 +19,21 @@
 
     <p class="text">{{ (page ~ '.description1') | trans(transOptions) }}</p>
 
-    <details>
-        <summary>
-            <span class="summary">{{ (page ~ '.whatisMentalCapacitySummary') | trans }}</span>
+    <details class="govuk-details" data-module="govuk-details">
+        <summary class="govuk-details__summary">
+            <span class="govuk-details__summary-text">{{ (page ~ '.whatisMentalCapacitySummary') | trans }}</span>
         </summary>
-        <div class="text opg-indented-block">
+        <div class="govuk-details__text">
             <p>{{ (page ~ '.whatisMentalCapacityPara1') | trans(transOptions) }}</p>
             <p>{{ (page ~ '.whatisMentalCapacityPara2') | trans(transOptions)|raw }}</p>
         </div>
     </details>
 
-    <details {% if '104' in report.type %}class="push--bottom"{% endif %}>
```

Comment:
I noticed this if statement has been removed and I just wanted to check if this was intentional.
I've noticed the `push--bottom` class was removed from other places, I'm assuming it was there to improve the look of the page and is no longer needed?

Reply:
Yeah - I *think* what was happening was the custom classes applied required a bit of margin to give the page some spacing but using the built-in GDS classes everything displays nicely with no spacing issues.